### ISSUE TITLE
Cirque trackpad won't work

### ISSUE THREAD

**Issue by CardboardMechanic** [zzeneg/stront#4]
I just built a ChocV Stront. Lights, keys, square display, and rotary encoder all work fine. I've got the trackpad on the right side, using the precompiled firmware,  using OSX (not Windows) and I can't get the track pad to move the mouse. Is there something extra I'm missing like installing a driver? Solder a jumper that wasn't mentioned? Thanks!

**Comment by zzeneg** [comment]
- did you check that the trackpad is connected correctly? e.g. VCC/GND on trackpad (there are testing pads on the bottom) have continuity to VCC/GND on the MCU?
- do you use trackpad in SPI (default) mode?
- which firmware exactly did you use?

**Comment by CardboardMechanic** [comment]
Thanks zzeneg! 
1) I've attached a photo to make sure I understand you. Are the "testing pads" what I've circled in the photo? If so, I've got continuity for most pins on the connector, but nothing when I probe RES and BL from the testing pads. All testing pads have good continuity to the MCU. 
2) Yes, there's a resistor in R1 on the trackpad, so it should be SPI from what I read in the datasheet. I'm using a 40mm flat trackpad. 
3) I used the file `firmware/qmk/stront_zzeneg-rect-flat.uf2`
![Cirque Diagnosis](https://github.com/zzeneg/stront/assets/163532/6296f60f-8f32-4c45-b69c-c5782ae19e90)

**Comment by zzeneg** [comment]
No, I meant pads on the cirque itself:
https://github.com/zzeneg/stront/blob/main/build-guide/choc/17.jpg
Connect trackpad to keyboard and test continuity between testing pads and MCU - first GND and VCC to verify that cable is correctly oriented and inserted, then pin 9 to SS, 10 to SCK, 11 to SI, 12 to SO (I'm not 100% sure on this mapping but just make that pin is connected to some pad 😄).

**Comment by CardboardMechanic** [comment]
Ah, yes, the pads on the cirque make sense. I assume the BTN1, BTN2, and BTN3 aren't wired to anything? 

Otherwise it looks like I have problems: 
1. DR and SS on the Cirque both go to pin 9 on the MCU
2. SDA and SCK on the Cirque both go to pin 10 on the MCU
3. SI and SCL on the Cirque both got to pin 11 on the MCU

**Comment by zzeneg** [comment]
2 and 3 are ok, this is how it should be (to support both SPI and I2C modes). However, I don't think that SS should be connected to DR so it may be the problem. I think you have shorted them underneath the FPC vertical connector (second from top pins on both sides). Try to clean and reflow/remove extra solder maybe?

BTNs should not be connected. And to double check, 12 is connected to SO, correct?

**Comment by CardboardMechanic** [comment]
I ended up going with the daughterboard ([this one on Amazon](https://a.co/d/1Zrr41h)). Using the pads on the back of the Cirque I figured out the right ribbon and orientation combination - I'm using the cable with blue on opposite sides and the connector on the daughterboard facing "down" toward the PCB. I had tried a daughterboard before, but had it upside down. Here's a picture showing what I landed on. Mouse works great now. Thanks for all the help @zzeneg!
![daughterboard](https://github.com/zzeneg/stront/assets/163532/fbf13b8e-f436-4211-99cd-f0a6b7f0f646)
